addons-reference.md

```diff
@@ -0,0 +1,58 @@
+This document provides a complete reference on addons in Anbox Cloud.
+
+<h2 id='heading--addon-reference-file-structure'>File structure</h2>
```

Comment:
I've started using
```
<a name="short-name"></a>
## Heading
```
to add an anchor. The advantage is that it's easier to change the heading level and to remember the anchor name, but the disadvantage is that it adds a bit of empty space.
Both should work though.